pillar/fluentd/reddit.sls

```diff
@@ -1,13 +1,8 @@
 {% from "fluentd/record_tagging.jinja" import record_tagging with context %}
+{% from "fluentd/tls_forward.jinja" import tls_forward with context %}
 
 fluentd:
-  overrides:
-    pkgs:
-      - ruby2.3
-      - ruby2.3-dev
-      - build-essential
```

Comment:
Any reason you're removing the ruby package name overrides?

Reply:
This came across as inconsistent as some files have it and others don't. When looking at the `map.jinja` file in the fluentd formula, those are listed under the Debian pkgs.